.eslintrc.json

```diff
@@ -143,7 +143,11 @@
         "react/no-deprecated": "error",
         "react/jsx-curly-brace-presence": [
             "error",
-            "never"
+            {
+                "props": "never",
+                "children": "never",
+                "propElementValues": "always"
```

Comment:
This is a new but it is as stated in the following issue https://github.com/yannickcr/eslint-plugin-react/issues/3094 not supported by all parser so we need to define it to always so our parsers do not have problems with it.